Src/Core/CaseStudy.Core/Contracts/IUnitOfWork/IHeaderFooterSettingsServices.cs

```diff
@@ -0,0 +1,20 @@
+﻿using CaseStudy.Core.Models;
+
+namespace CaseStudy.Core.Contracts.IUnitOfWork
+{
+    public interface IHeaderFooterSettingsServices
+    {
+        public Task<IEnumerable<HeaderAndFooterSettings>> GetHeaderFooterSettings();
```

Comment:
access modifier public is not needed for interface members, it's by default public

Reply:
okay